devstack/local.conf.nfp

```diff
@@ -16,28 +16,24 @@ SERVICE_TOKEN=admin
 
 # NFP Service
 ENABLE_NFP=True
-# Edit NFP_DEVSTACK_MODE with either 'base' or 'advanced' or 'enterprise'
+# Edit NFP_DEVSTACK_MODE with either 'base' or 'advanced'
 [[ $ENABLE_NFP = True ]] && NFP_DEVSTACK_MODE=base
-[[ $NFP_DEVSTACK_MODE != base ]] && enable_plugin neutron-vpnaas https://git.openstack.org/openstack/neutron-vpnaas stable/mitaka
-[[ $NFP_DEVSTACK_MODE != base ]] && enable_plugin ceilometer https://git.openstack.org/openstack/ceilometer.git stable/mitaka
-
+[[ $NFP_DEVSTACK_MODE = advanced ]] && enable_plugin neutron-vpnaas https://git.openstack.org/openstack/neutron-vpnaas stable/mitaka
+[[ $NFP_DEVSTACK_MODE = advanced ]] && enable_plugin ceilometer https://git.openstack.org/openstack/ceilometer.git stable/mitaka
 if [[ $NFP_DEVSTACK_MODE = base ]]; then
     GBPSERVICE_REPO=https://git.openstack.org/openstack/group-based-policy
     GBPSERVICE_BRANCH=master
-elif [[ $NFP_DEVSTACK_MODE = advanced ]]; then
+else
     # Openstack repo
     #GBPSERVICE_REPO=https://git.openstack.org/openstack/group-based-policy
     #GBPSERVICE_BRANCH=refs/changes/05/335405/27
     # Oneconvergence repo
     GBPSERVICE_REPO=https://github.com/oneconvergence/group-based-policy.git
     GBPSERVICE_BRANCH=mitaka_21st_march_base
-else
-    GBPSERVICE_REPO=https://github.com/oneconvergence/group-based-policy.git
-    GBPSERVICE_BRANCH=mitaka_21st_march_base
 fi
 enable_plugin gbp $GBPSERVICE_REPO $GBPSERVICE_BRANCH
 
-if [[ $NFP_DEVSTACK_MODE != base ]]; then
+if [[ $NFP_DEVSTACK_MODE = advanced ]]; then
```

Comment:
Couldn't find haproxy image path. Instead, haproxy V2 path is provided. We don't need to provide haproxy v2 as first step.